roles/openldap/meta/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+dependencies:
+  - role: logging
```

Comment:
logging-conf role takes params, the logging role does not. logging-conf role sets up logging with logstash, logging setups rsyslog.
logging-conf has dependency on logging, we are not setting up logging for logstash for now, its only to log to a local file using rsyslog.